components/build/README.md

```diff
@@ -1,113 +0,0 @@
-
-# App Studio/HACBS Build System
```

Comment:
imo this readme file contains useful information - is none of it going to be needed in a future?

Reply:
hmm, probably I could split it as well

Reply:
just moved to build-service for now.